### ISSUE TITLE
esites: cleanup Puerto Rico records

### ISSUE THREAD

**Issue by rondlg** [fieldmuseum/EMu-Documentation#1128]
### Discussed in https://github.com/fieldmuseum/EMu-Documentation/discussions/1084

<div type='discussions-op-text'>

<sup>Originally posted by **rondlg** June  7, 2024</sup>
### Standard record for Puerto Rico
- [Existing standard hierarchy for Puerto Rico](https://github.com/fieldmuseum/EMu-Documentation/issues/90)

**DECISION: Continue with this as a standard** 

**TASK**: [Clean up Puerto Rico Records](https://github.com/fieldmuseum/EMu-Documentation/issues/1128)

_Originally posted by @rondlg in https://github.com/fieldmuseum/EMu-Documentation/discussions/1084#discussioncomment-9799405_
</div>

**Comment by rondlg** [comment]
@zkachian here's the originating PR issue.

**Comment by rondlg** [comment]
@zkachian Do you have what you need for the PR cleanup?

**Comment by zkachian** [comment]
I don't think so. It looks like the standards hierarchy doc says that for PR it should be: Country/PD1 = Puerto Rico; Sovereignty = United States. But I remember in our last conversation with Chris M and Chris P that we talked about having it as: PD2 = Puerto Rico; Rank = Territory; Sovereignty = United States. I'm not sure if we ever finalized a decision, so I'm not sure which standard to follow.
